main.s

```diff
@@ -118,9 +132,7 @@ ENDR
 SECTION "Support Routines",ROM0
 
 WaitVBlank::
-  ldh a, [rLY] ; Get current scanline
-  cp SCRN_Y ; Are we in v-blank yet?
-  jr nz, WaitVBlank ; If A-91 != 0 then loop
+  halt

```

Comment:
Because all other interrupts are disabled.

https://github.com/ISSOtm/GBZ80-ASM-Gameboy-Project/blob/359ffd290cd7a1f97f41411b3523363402325d7a/main.s#L192-L194